python/example/accuracy_pitfalls.ipynb

```diff
@@ -0,0 +1,338 @@
+    "\n",
+    "Note that our diagnostic testing of $\\tilde{D}$ in the code above is not trivial in this case. In the first example, we knew that clamp/resize did not change the underlying data, so we could predict exactly the data on which the DP mean would actually be calculated. This will not be true for the following examples, so we will simulate finding the true underlying mean by releasing an extra DP mean with very high epsilon."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 50,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy interval (with accuracy value 0.2853) contains the true mean on D_tilde with probability 0.4725\n",
+      "Accuracy interval (with accuracy value 0.2853) contains the true mean on D with probability 0.4725\n"
+     ]
+    }
+   ],
+   "source": [
+    "# establish extra information for this simulation\n",
+    "age_lower_bound = 0.\n",
+    "age_upper_bound = 100.\n",
+    "data_size = 1050\n",
+    "\n",
+    "n_sims = 1_000\n",
+    "true_D_tilde_mean_ages = []\n",
+    "dp_D_tilde_mean_ages = []\n",
+    "\n",
+    "D_tilde = np.clip(D.age, age_bounds[0], age_bounds[1])\n",
+    "D_tilde_mean_age = np.mean(D_tilde)\n",
+    "\n",
+    "def check_scale(scale, preprocessor, dataset_distance, epsilon):\n",
+    "    return (preprocessor >> make_base_laplace(scale)).check(dataset_distance, epsilon)\n",
+    "\n",
+    "preprocessor = (\n",
+    "    # Convert data into a dataframe of string columns\n",
+    "    make_split_dataframe(separator=\",\", col_names=var_names) >>\n",
+    "\n",
+    "    # Selects a column of df, Vec<str>\n",
+    "    make_select_column(key=\"age\", TOA=str) >>\n",
+    "\n",
+    "    # Cast the column as Vec<float>\n",
+    "    make_cast_inherent(TIA=str, TOA=float) >>\n",
+    "\n",
+    "    # TODO: we need to impute missing values sampled from uniform, not a constant (see line 168)\n",
+    "    make_impute_uniform_float(bounds=age_bounds) >>\n",
+    "\n",
+    "    # Clamp age values\n",
+    "    make_clamp(bounds=age_bounds) >>\n",
+    "\n",
+    "    # Resize the dataset to length `data_size`.\n",
+    "    #     If there are fewer than `data_size` rows in the data, fill with a constant of 20.\n",
+    "    #     If there are more than `data_size` rows in the data, only keep `count_release` rows\n",
+    "    make_bounded_resize(size=data_size, bounds=age_bounds, constant=20.) >>\n",
+    "\n",
+    "    # Compute the mean\n",
+    "    make_sized_bounded_mean(size=data_size, bounds=age_bounds)\n",
+    ")\n",
+    "\n",
+    "scale = binary_search(lambda s: check_scale(s, preprocessor, 1, epsilon), bounds=(0.0, 1000.0))\n",
+    "\n",
+    "preprocessor = preprocessor >> make_base_laplace(scale)\n",
```

Comment:
Same argument as before- use the specialized function, rename the preprocessor variable once you've chained with a measurement.